src/actions/clipboard.ts

```diff
@@ -0,0 +1,373 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ContextMenuRegistry,
+  Gesture,
+  ShortcutRegistry,
+  utils as BlocklyUtils,
```

Comment:
Since this is a new name I'd probably stick with styleguide advice re: case:

```suggestion
  utils as blocklyUtils,
```

Reply:
Done